packages/vc-verification/src/resolver/data-aggregator.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * An Interface for Data aggreator to cache responses from Blockchain and SSI-Hub
+ */
+export interface IDataAggregator {
+  /**
+   * Stores role credential
+   * @param did user DID
+   * @param role namespace
+   * @param data Credential of the user for the namespace
+   */
+  setRoleCredential(did: string, role: string, data: any): void;
```

Comment:
Can we set exact type of data?
```suggestion
  setRoleCredential(did: string, role: string, data: RoleEIP191JWT | VerifiableCredential<RoleCredentialSubject>): void;
```